src/gui/touchscreengui.cpp

```diff
@@ -892,12 +892,12 @@ void TouchScreenGUI::translateEvent(const SEvent &event)
 					m_move_id                  = event.TouchInput.ID;
 					m_move_has_really_moved    = false;
 					m_move_downtime            = porting::getTimeMs();
+					m_move_sent_as_mouse_event = false;
 					if (m_use_crosshair || m_camera_mode == CAMERA_MODE_THIRD) {
```

Comment:
Braces aren't required as there's only one line of code within the if and the else blocks.

Reply:
Should I also fix for other lines?

Reply:
It'd be great if you could fix this for all touched lines.

Reply:
Is 8ec9524e456bcaa73b2d79f74ccdb11bfa7bb8b0 enough? I can't find other touched lines that should be fixed. Did you mean also neighbouring lines?

Reply:
I didn't check whether there were any other lines that needed fixing; I assumed there were, when you said "Should I also fix this for other lines?". But looking at the code, I don't see any other line that adds braces to single-line blocks. So please ignore my last request. :)